src/modules/Builder/pages/BuilderApplet/BuilderApplet.tsx

```diff
@@ -61,17 +64,21 @@ export const BuilderApplet = () => {
   } = methods;
 
   useEffect(() => {
-    if (loadingStatus === 'success' && !isNewApplet) {
+    if (location.state?.isFromLibrary) setIsFromLibrary(true);
+  }, [location.state]);
```

Comment:
per my understanding you have to check particular value not the `state` object ? 
```suggestion
  }, [location.state?.isFromLibrary]);
```

Reply:
fixed